modules/server/src/schema/configQuery.ts

```diff
@@ -75,13 +75,14 @@ export default `
   ${FacetsConfigTypeDefs}
   ${MatchBoxConfigTypeDefs}
   ${TableConfigTypeDefs}
-
+  
   type ConfigsWithState {
     facets: FacetsConfig
     downloads: DownloadsConfig
     extended(fieldNames: [String]): JSON
     matchbox: [MatchBoxMapping] # we may want to rethink this one later
     table: TableConfig
+    charts: JSON
```

Comment:
basic JSON return type for now, as this is minimal POC, "just return hardcoded content" implementation

Reply:
please alphabetize these too? (I realize "facets" is out of place, my mistake)